gm4_area_markers/pack.mcmeta

```diff
@@ -0,0 +1,24 @@
+{
+  "pack":{
+    "pack_format":0,
+    "description":"A Gamemode 4 Module"
+  },
+  "module_name":"Area Markers",
+  "module_id":"area_markers",
+  "site_description":"Set up floating messages with a simple name tag!",
+  "site_categories":[],
+  "video_link":"",
+  "wiki_link":"",
+  "required_modules":[
+    "custom_crafters"
+  ],
+  "recommended_modules":[],
+  "credits":{
+	"Creator":[
+	  ["Bunnygamers","https://twitter.com/BunnygamersMC"]
+	],
+    "Code mostlty just copied from":[
```

Comment:
I don't think the site has an option for this.

Reply:
> I don't think the site has an option for this.

Actually you can put anything in the credits object

Reply:
> > I don't think the site has an option for this.
> 
> Actually you can put anything in the credits object

Ooh, cool, but it's worded kinda weird IMHO.